README.md

```diff
@@ -5,7 +5,7 @@ A public collection of useful scripts to help manage the Mellanox BlueField
 SoC.
 
 Overview of each file:
-- **bfauxpwr** Config ACPI daemon to handle AUX power mode event.
```

Comment:
might need another commit in ezchip repo as well to fix the recipe file, or else master branch could fail.

Reply:
whoops I will do it now.